lib/miq_automation_engine/engine/miq_ae_engine/drb_remote_invoker.rb

```diff
@@ -45,20 +45,20 @@ def teardown
       global_id_conv = drb_server.config[:idconv]
       drb_server.stop_service
       self.drb_server = nil
+
       # This hack was done to prevent ruby from leaking the
       # TimerIdConv thread.
       # https://bugs.ruby-lang.org/issues/12342 (has been fixed in ruby 2.4.0 preview 1)
       # also fixed in ruby_2_3 branch for the 2.3.2 release: https://github.com/ruby/ruby/commit/c20b07d5357d7cb73226b149431a658cde54a697
-      thread = global_id_conv
-               .try(:instance_variable_get, '@holder')
-               .try(:instance_variable_get, '@keeper')
-      if RUBY_VERSION > "2.3.1"
-        warn "Remove me: #{__FILE__}:#{__LINE__} and my test. Ruby 2.3.2+ should not be creating timer threads."
```

Comment:
Would maybe some kind of `.travis.yml` parser that checks the supported ruby versions be a good way to handle this?  Might be over-engineering things a bit, but something to consider so we aren't forgetting to remove this in the future.  This of course assumes that the `.travis.yml` is consistent here and in downstream as a source of truth for supported ruby versions.

Otherwise I am at a loss on how we could do this.